### DISCUSSION TITLE
Library not loaded: '@rpath/libuuid.1.dylib'

### DISCUSSION THREAD

**Discussion by svenborgstrom** [isce-framework/isce2#580] [Q&A]
Good morning,
trying to use mdx.py (e.g. mdx.py 20220703.slc) I get the error message in the subject:

...
Running: mdx 20220703.slc -c8 -s 13829 
dyld[1621]: Library not loaded: '@rpath/libuuid.1.dylib'
  Referenced from: '/Users/sven/tools/miniconda3/envs/insar/lib/libXm.4.dylib'
  Reason: tried: '/Users/sven/tools/miniconda3/envs/insar/lib/libuuid.1.dylib' (no such file), '/Users/sven/tools/miniconda3/envs/insar/lib/libuuid.1.dylib' (no such file), '/Users/sven/tools/miniconda3/envs/insar/lib/python3.9/site-packages/isce/bin/../../../../libuuid.1.dylib' (no such file), '/Users/sven/tools/miniconda3/envs/insar/lib/python3.9/site-packages/isce/bin/../../../../libuuid.1.dylib' (no such file), '/usr/local/lib/libuuid.1.dylib' (no such file), '/usr/lib/libuuid.1.dylib' (no such file)

Something related to a wrong PATH and/or missing file/library?
Any suggestions would be appreciated.
Best,
Sven.

**Comment by EJFielding** [comment]
Hi Sven,

How did you install ISCE in your conda environment?  It looks like your environment might be not quite correct. I would recommend deleting it and recreating it. The `conda install isce2` command should work to install version 2.6.0 at this time. Anaconda or Miniconda do the management of your PATH and PYTHONPATH, so you should normally not change those.

**Reply by svenborgstrom** [reply]
Hi Eric,
thanks for reply.
I installed ISCE jointly with MintPy, including it in a common file (requirements.txt) with many other python packages (conda install -c conda-forge --file ~/tools/MintPy/requirements.txt) in order to have all in a single conda environment.
Following your suggestion I removed then reinstalled it but I always get the same error message.
My feeling, looking on Google, is this is something related to the Xcode settings.
B.t.w. for the moment I'll use the "view.py" tool from MintPy temporarily in replacement.
Sven.

**Reply by EJFielding** [reply]
Hi Sven,
I don't know how Anaconda works with the Xcode libraries. I did find that `libuuid` is a library you can install with `conda`. See: https://anaconda.org/conda-forge/libuuid
